server/datastore/mysql/migrations/tables/20231024105026_AddWindowsMDMTables.go

```diff
@@ -0,0 +1,111 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231024105026, Down_20231024105026)
+}
+
+func Up_20231024105026(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+-- This table contains the commands, which may target multiple devices.
+CREATE TABLE windows_mdm_commands (
+	-- managed and generated by Fleet, and used as CmdID in the MSMDM messages.
+	command_uuid     VARCHAR(127) NOT NULL,
+	-- the raw XML of the command
+	raw_command      MEDIUMTEXT NOT NULL,
+	-- whether the command was generated from the system or from a user
+	system_origin    TINYINT(1) NOT NULL DEFAULT FALSE,
```

Comment:
in the spirit of removing this we don't need now, should we get rid of this? AFAIK there's no requirement to do this distinction.

Reply:
Good call. The only concern is that I don't think we'd be able to backfill this with 100% confidence if we added it later (unless we never send `Exec` commands system-initiated), but given that we may never need this, I'll go ahead and remove it.

Reply:
I don't have a strong opinion on this tbh

Reply:
I think that's a small/unlikely concern, I went ahead and removed the column.